src/lib.rs

```diff
@@ -1620,6 +1621,32 @@ fn weight_callable(
     }
 }
 
+#[pyfunction(normalized = "true", endpoints = "false")]
```

Comment:
This will need a docstring (with `///`) in the sphinx rst format. The pyfunction macro will set this string to `__doc__` on the  python function it generates. Also since these are lib functions we'll need to add them to the `docs/source/api.rst` so they get included in the docs build.